.gitignore

```diff
@@ -30,3 +30,4 @@ src/main/docs/resources/css/*.css
 src/main/docs/resources/js/*.js
 src/main/docs/resources/style/*.html
 src/main/docs/resources/img/micronaut-logo-white.svg
+**/.micronaut/test-resources/
```

Comment:
Some test-resources files started showing up recently, it's safe to ignore these paths.

Reply:
What files are those?

Reply:
It's indeed safe to ignore these: the Gradle plugin behavior has been aligned to that of the Maven plugin which creates `.micronaut` directories. This also has the side effect of allowing to use test resources even if not delegating builds to Gradle.